amazon/ionbenchmark/json_load_dump.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+
+class JsonLoadDump:
+    def __init__(self):
+        pass
+
+    def load(self, fp):
+        while True:
+            line = fp.readline()
+            if line == '':
+                return
+            yield json.loads(line)
+
+    def loads(self, s):
+        return json.loads(s)
```

Comment:
so loads can only handle single values?

Reply:
`io_type=buffer` currently doesn't handle multiple top-level use case, I opened an issue for that - https://github.com/amazon-ion/ion-python/issues/325

Reply:
Ok. At first blush it seems like just making multiple TLVs there would be simpler than creating an issue and having divergent code paths, but we need to get this merged.